### DISCUSSION TITLE
Query rendered features at point not returning correct value

### DISCUSSION THREAD

**Discussion by kevinmanncito** [rnmapbox/maps#2600] [Q&A] [answered]
The call to `queryRenderedFeaturesAtPoint` is only ever returning this object for me:

```
{"_h": 0, "_i": 0, "_j": null, "_k": null}
```

Here is my code (from the `onPress` of the `MapView` component):

```
onPress={async (feature: GeoJSON.Feature) => {

  const features = mapRef.current.queryRenderedFeaturesAtPoint(
    [feature.properties.screenPointX, feature.properties.screenPointY]
  );
  console.log('features', features);

}}
```

* I've tested on android and ios (same result)
* I've put break points in the swift code and confirmed the object being passed into the `resolver` function is correct.
* I'm using `"rnmapbox/maps#main"` for the package.json dependency
* I'm using mapbox v10 for the underlying sdk

Has anyone seen this before? Thanks!

**Comment by mfazekas** [comment, marked as answer]
@kevinmanncito it's an async function and you're missing an await

**Reply by kevinmanncito** [reply]
How did I miss that!! Thanks!
